flight/targets/naze32/fw/pios_board.c

```diff
@@ -424,6 +335,110 @@ void PIOS_Board_Init(void) {
 	//I2C is slow, sensor init as well, reset watchdog to prevent reset here
 	PIOS_WDG_Clear();
 
+	/* Configure the rcvr port */
+	// Note: must come after the MPU init so we know which board rev
+
+	switch (hw_rcvrport) {
+
+	case HWNAZE_RCVRPORT_DISABLED:
+		break;
+
+	case HWNAZE_RCVRPORT_PWM:
+		PIOS_HAL_ConfigurePort(HWSHARED_PORTTYPES_PWM,  // port type protocol
+				NULL,                                   // usart_port_cfg
+				NULL,                                   // com_driver
+				NULL,                                   // i2c_id
+				NULL,                                   // i2c_cfg
+				NULL,                                   // ppm_cfg
+				&pios_pwm_cfg,                          // pwm_cfg
+				PIOS_LED_ALARM,                         // led_id
+				NULL,                                   // dsm_cfg
+				0,                                      // dsm_mode
+				NULL);                                  // sbus_cfg
+		break;
+
+	case HWNAZE_RCVRPORT_PPMSERIAL:
+	case HWNAZE_RCVRPORT_SERIAL:
+		{
+			uint8_t hw_rcvrserial;
+			HwNazeRcvrSerialGet(&hw_rcvrserial);
+
+			HwNazeDSMxModeOptions hw_dsmx_mode;
```

Comment:
This switch is something other targets are missing.  We should figure out how to factor this into PIOS_HAL.